### DISCUSSION TITLE
C++

### DISCUSSION THREAD

**Discussion by rockoguido** [Mi-V-Soft-RISC-V/miv-rv32-documentation#29] [Bare Metal Embedded Software]
Looking for any C++ example actually targeting a SmartFusion2 SoC.   I could not locate a discussion area for SmartFusion2 SoC but was hoping posting here could lead to a link to SmartFusion 2 targeted examples.  
Thx!
Dave

**Comment by CLappin** [comment]
Hi Dave, 
I'll ask internally and come back to you. 
The discussion areas are generally aimed at RISC-V on this and the PolarFire SoC discussion forums. 

Thanks,
Ciaran

**Comment by rockoguido** [comment]
Hi Ciaran,

Thanks for the response, very much appreciated.    Yes for the SF2 just not sure if such a forum exists.  So, I went with what I know.
Any insight would be awesome

Thx!

Dave Campanella
Principal Embedded Solutions Engineer / FAE
Microchip Technology Inc.
165 Technology Dr
Irvine, CA 92618
+1 925 989 0569 (mobile)
+1 760 304 0172 (office)
***@***.******@***.***>

[Text  Description automatically generated]
<http://www.microchip.com/emasters>Visit our new learning site at www.microchip.com/mu<http://www.microchip.com/mu>
